app/models/lockstep/feature_flag.rb

```diff
@@ -0,0 +1,12 @@
+class Lockstep::FeatureFlag < Lockstep::ApiRecord
+  self.model_name_uri = "v1/feature-flags"
+  self.query_path=""
+  load_schema(Schema::FeatureFlag)
+
+  # Create method to override conventional create in lockstep/api_record.rb for sending attrs as an enclosed hash/object
+  def create
```

Comment:
for this method also we can add a Rspec with positive/ negative scenarios

Reply:
Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/22/commits/e5385986fc8af5e4ef9876da9898a30f618b2ac2)